pkg/instance/instance_test.go

```diff
@@ -97,9 +101,10 @@ func TestExecprogCmd(t *testing.T) {
 	flagCollide := flags.Bool("collide", false, "collide syscalls to provoke data races")
 	flagSignal := flags.Bool("cover", false, "collect feedback signals (coverage)")
 	flagSandbox := flags.String("sandbox", "none", "sandbox for fuzzing (none/setuid/namespace)")
+	flagSandboxArg := flags.Int("sandbox_arg", 0, "Android uses to specify account for sandbox.")
 	flagSlowdown := flags.Int("slowdown", 1, "")
 	cmdLine := ExecprogCmd(os.Args[0], "/myexecutor", targets.FreeBSD, targets.I386,
-		"namespace", true, false, true, 7, 2, 3, true, 10, "myprog")
+		"namespace", 3, true, false, true, 7, 2, 3, true, 10, "myprog")
```

Comment:
Same. Lets use named constant.

Reply:
I commented previously. The value '3' should be viewed in the same way as values "7, 2, 3, ...., 10" in this arguments list.